packages/anon-aadhaar-react/src/verifySignature.ts

```diff
@@ -0,0 +1,67 @@
+import { convertBigIntToByteArray, decompressByteArray } from 'anon-aadhaar-pcd'
```

Comment:
Should we move it to pcd module? I also not sure

Reply:
Yes
So this one can't be included in the pcd package as it is browser-specific, it's using the subtle crypto API by querying it from the window object, so it can't be run in Nodejs.
Do we need the equivalent for pcd package?

Reply:
I remember crypto-js also support verify function. However, we can do it in other PR.

Reply:
I wasn't able to find any method that verifies RSA signatures in crypto-js, that's why I used subtle from window.